src/System.Threading.Tasks/tests/System.Runtime.CompilerServices/TaskAwaiterTests.cs

```diff
@@ -170,6 +171,83 @@ public async Task Await_TaskCompletesOnNonDefaultScheduler_ContinuesOnDefaultSch
             });
         }
 
+        public static IEnumerable<object[]> Await_MultipleAwaits_FirstCompletesAccordingToOptions_RestCompleteAsynchronously_MemberData()
+        {
+            foreach (int numContinuations in new[] { 1, 2, 5 })
+                foreach (bool runContinuationsAsynchronously in new[] { false, true })
+                    foreach (bool valueTask in new[] { false, true })
+                        foreach (object scheduler in new object[] { null, new QUWITaskScheduler(), new ValidateCorrectContextSynchronizationContext() })
+                            yield return new object[] { numContinuations, runContinuationsAsynchronously, valueTask, scheduler };
+        }
+
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "All await continuations complete synchronously on netfx.")]
```

Comment:
Do we still need to add this line to skip tests on .NET Framework? I thought @ViktorHofer changed everything to skip .NET Framework test run unless the binary/package ships OOB on .NET Framework.

Reply:
> I thought @ViktorHofer changed everything to skip .NET Framework test run unless the binary/package ships OOB on .NET Framework.

Really?  @ViktorHofer is that true?  Why did we do that?

Reply:
See: #35088 and PR #37972

>Remove netfx tfm, NetFramework SkipOnTargetFramework and IsFullFramework conditions where we are not live-building or shipping netfx assets. I left some netfx comments in cases where context is helpful.

Reply:
So @ViktorHofer, how are we validating that .NET Core behavior is compatible with .NET Framework behavior?

Reply:
I raised this concern in #35088.  But the consensus was  not to run tests in CI purely to validate .NET Framework behavior, but only to validate it when we ship separate binaries for it from CoreFx.

Reply:
@stephentoub see https://github.com/dotnet/corefx/issues/35088 where we discussed this. Also we didn't remove the netfx configuration at places like BinaryFormatter. I applied @jkotas suggestion to remove netfx code paths if the assembly does not have a shipping netfx  configurauibn

Reply:
The answer I was told by @danmosemsft was something like "if it's anything where we're concerned, just add the test configuration back" (possibly with a "do this manually, then call it a day" rider).